highfidelity/hifi
# Removed rescaling of dragged-dropped zipped models (bug 12599).

# Description
This "bug" was causing large models that have been dropped into Interface to be very small.  This was actually a feature that was implemented for specific models that were in cm/mm instead of metres.  The effect was that the models were way too big.

PLEASE READ - https://highfidelity.manuscript.com/f/cases/12599/


# Testing
Use this file for testing:  https://hifi-content.s3.amazonaws.com/elisalj/models/whale.zip

1. Drag and drop zip file: size is correct (a 30m blue whale (note that the model is not blue).
2. Find the unzipped file in the asset browser and add to world: same size
3. Add via Model in Create menu (using https://hifi-content.s3.amazonaws.com/elisalj/models/whale/whale.obj): comes in same size.
